plugins/processors/k8sdecorator/stores/utils.go

```diff
@@ -30,11 +31,12 @@ func createContainerKeyFromMetric(tags map[string]string) string {
 	return k8sutil.CreateContainerKey(namespace, podName, containerName)
 }
 
-const (
-	// kubeAllowedStringAlphaNums holds the characters allowed in replicaset names from as parent deployment
+var (
+	// deploymentUnallowedRegExp holds the characters allowed in replicaset names from as parent deployment
 	// https://github.com/kubernetes/apimachinery/blob/master/pkg/util/rand/rand.go#L83
-	kubeAllowedStringAlphaNums = "bcdfghjklmnpqrstvwxz2456789"
-	cronJobAllowedString       = "0123456789"
+	deploymentUnallowedRegExp = regexp.MustCompile(`[^b-hj-np-tv-xz-z2-24-9]+`)
+	// cronJobUnallowedRegexp ensures the characters in cron job name are only numbers.
+	cronJobUnallowedRegexp = regexp.MustCompile(`[^0-9]+`)
```

Comment:
BTW, this regex could be simplified to:
```go
cronJobUnallowedRegexp = regexp.MustCompile(`\D+`)
```

Reply:
Nice catch. Fixed it in the newest commit.